components/expenses/CreateExpenseForm.js

```diff
@@ -153,9 +164,7 @@ class CreateExpenseForm extends React.Component {
   }
 
   async onSubmit(e) {
-    if (e) {
-      e.preventDefault();
-    }
+    e && e.preventDefault();
```

Comment:
While we're at it, it's best to not change/refactor unrelated code and as a team we prefer the previous syntax.

Reply:
Reverted @znarf 😄